### DISCUSSION TITLE
Set permanent DNS nameservers

### DISCUSSION THREAD

**Discussion by malikshi** [trinib/Adguard-Wireguard-Unbound-Cloudflare#60] [Q&A] [answered]
the new wiki about Set permanent DNS nameservers, is it won't getting override whenever WAN interface up/restarted?

**Comment by trinib** [comment, marked as answer]
@malikshi once dns is set it will remain after system reboot. I added this guide cause **sometimes** for some strange reason after installing unbound which sets 127.0.0.1 in  `/etc/resolv.conf` and removes the default ones, local host is unable to receive internet access. It does this sometime and i did not really investigate exactly why. 

This method with resolv package is just a easy way to add more dns nameservers or your home network dns/gateway to get host reachable to internet if it happens to someone.

**Reply by malikshi** [reply]
oke latest wiki work properly :)
